deploy/crds/openebs_v1alpha1_blockdeviceclaim_cr.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: example-blockdeviceclaim
 spec:
   hostName: "" # hostname of the node from which you want to get a BD
+  nodeAttributes:
+    nodeName: "" # node name of the k8s node. Output from `kubectl get nodes`
```

Comment:
can we also include the `hostname` here. It will be a duplicate, but we can probably deprecate the hostName from above for newer versions. Btw, is spec.hostName optional?

Reply:
we can include it. Currently the same `NodeAttribute` struct is used for both BD and BDC. BD does not need a `hostname` in its node-attributes as it is present in the label. 
So, will make a new type for the BDC-Node attributes, it will help in deprecating the `hostname` field.

Also, with this PR hostname is optional.